libarchive/types/archive.py

```diff
@@ -1,5 +1,10 @@
 from ctypes import *
 
+try:
+    from ctypes import c_ssize_t
```

Comment:
The source-code for [ctypes](https://github.com/python/cpython/blob/master/Lib/ctypes/__init__.py#L456) shows how these types are defined:

```python
if sizeof(c_uint) == sizeof(c_void_p):
    c_size_t = c_uint
    c_ssize_t = c_int
elif sizeof(c_ulong) == sizeof(c_void_p):
    c_size_t = c_ulong
    c_ssize_t = c_long
elif sizeof(c_ulonglong) == sizeof(c_void_p):
    c_size_t = c_ulonglong
    c_ssize_t = c_longlong
```

Can you update to the same thing?